README.md

```diff
@@ -15,21 +17,34 @@ you need to run ```source /opt/ros/melodic/setup.bash``` in your current shell f
 
 On the **first usage**, run ```./setup.sh``` to create virtual environment and install requirements.
 
-1. **Run** ```./extract.sh <PATH_TO_BAG> .. <PATH_TO_BAG>``` with **paths to your .bag files** as arguments
-    - Make sure topics for all message types in ```./extract.sh``` match your topics (you can update them for your needs)
+1. Setup **topic names** for extraction in ```./extract.conf``` file
+2. **Run** ```./extract.sh <PATH_TO_BAG> (optional)<PATH_TO_VIDEO>``` with **path to your .bag file** 
+    and **path to smartphone video** from OpenCamera Sensors as arguments
+     *(note: directory with meta information about video, e.g. ```20210119_233624```, should be in the same path as video)*.
 3. **Data is saved** to the subdirectories of ```./<YOUR_BAG_NAME>``` directories (subdirectory name = topic name with ```/``` replaced with ```_```):
-    - ```{camera_topic_name}``` - images from camera with timestamps as filenames
+    - ```{camera_topic_name}``` - images from camera with timestamps as filenames in ```jpeg``` format
+    - ```{depth_camera_topic_name}``` - depth images with timestamps as filenames in ```tiff``` format 
     - ```{poincloud_topic_name}``` - point cloud files with timestamps as filename
     - ```{time_ref_topic_name}``` - time reference files, format:
         ```
-        <timestamp, time_ref>
+        <sequence_number, timestamp, time_ref>
         ```
     - ```{imu_topic_name}``` - ```csv``` files, format:
         ```
         <timestamp, ox, oy, oz, ax, ay, az, temperature>
         ```
         *where ox, oy, oz - angular velocity; ax, ay, az - linear acceleration*
+    - ```_s10_video_frame``` - extracted smartphone video frames with timestamps as filename
```

Comment:
What are the reasons for underscore in the beginning? Also, I'm not sure that we would like to specify the smartphone model directly in the folder name, because there could be different smartphones used, maybe we need to add meta-info in the same folder in recording app?

Reply:
Another option is to extract model meta-info from the video itself, possibly also with ffmpeg

Reply:
It turns out that smartphone model metadata isn't always available, so I'll leave a general name (```smartphone_video_frames```) for now and move to issues